arex-schedule-web-api/src/main/java/com/arextest/schedule/dao/mongodb/ReplayCompareResultRepositoryImpl.java

```diff
@@ -35,6 +45,30 @@ public boolean save(ReplayCompareResult item) {
         return true;
     }
 
+    public boolean upsert(List<ReplayCompareResult> itemList) {
+        List<ReplayCompareResultCollection> replayCompareResultCollections = itemList.stream()
+                .map(ReplayCompareResultConverter.INSTANCE::daoFromBo).collect(Collectors.toList());
+
+        BulkOperations bulkOps = mongoTemplate.bulkOps(BulkOperations.BulkMode.UNORDERED, ReplayCompareResultCollection.class);
+
+        List<Pair<Query, Update>> upserts = new ArrayList<>();
+        for (ReplayCompareResultCollection replayCompareResult : replayCompareResultCollections) {
+            Query query = new Query();
```

Comment:
Unable to uniquely locate compareresult, the missing of operationName

Reply:
Delete by recordId&planItemId first, then save list.